### DISCUSSION TITLE
Allow ignoring/skipping pruning of orphan packages

### DISCUSSION THREAD

**Discussion by star-szr** [CyberShadow/aconfmgr#198] [Ideas]
Thank you for this great tool!

I am not using aconfmgr to manage packages, using this technique: https://github.com/CyberShadow/aconfmgr/wiki/Ignore-all-packages

However when running `aconfmgr apply` I still get a prompt to remove orphan packages when there are orphan packages detected. If I say no, then aconfmgr halts with a fatal error:

```
:::::: Fatal error! Stack trace:
::::::: /usr/lib/aconfmgr/apply.bash:316 [AconfApply]
::::::: /usr/lib/aconfmgr/main.bash:188 [Main]
::::::: /usr/lib/aconfmgr/main.bash:205 [source]
::::::: /usr/bin/aconfmgr:26 [main]
```

If I run with the `--paranoid` flag, there is a slightly different result `User abort`, but it still doesn't proceed to the later steps.

I would be interested in one or both of the following:

1. Some kind of option - could be similar to the wiki link above - to skip pruning of orphan packages. Looking quickly at the code I didn't see any way of implementing this without a change to aconfmgr.
2. Continue running even if the user says no to pruning orphan packages.

**Comment by CyberShadow** [comment]
Thanks, I believe this should be fixed as of 1195f3a1b8173bb8ef7d6fd6c18ffb1570e74a4e.

**Reply by star-szr** [reply]
Confirmed, thank you!
